programming_examples/basic/packet_switch/host/host.cpp

```diff
@@ -0,0 +1,131 @@
+//===- test.cpp -------------------------------------------000---*- C++ -*-===//
```

Comment:
```suggestion
//===- test.cpp -------------------------------------------------*- C++ -*-===//
```

Reply:
Good news: the LLVM coding style changed recently about these lines and now just
```suggestion
//===----------------------------------------------------------------------===//
```
is enough.